reco_utils/dataset/wikidata.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import requests
+
+def find_wikidataID(name):
+    """Find the entity ID in wikidata from a title string.
+    Args:
+        name (str): A string with a wikidata page title.
+    Returns:
+        entityID: wikidata entityID corresponding to the title string. 
+                  'entityNotFound' will be returned if no page is found
+    """
+    url = "https://en.wikipedia.org/w/api.php?action=query&prop=pageprops&format=json&titles="
+    r = requests.get(url+name.replace(" ", "%20"))
+    entityID = list(r.json()["query"]["pages"].keys())[0]
+    try:
+        entityID = r.json()["query"]["pages"][entityID]["pageprops"]["wikibase_item"]
+    except:
+        entityID = "entityNotFound"
+    return entityID
+
+def query_entity_links(entityID):
+    """Query all linked pages from a wikidata entityID
+    Args:
+        entityID (str): A wikidata page ID.
+    Returns:
+        data (json): dictionary with linked pages.
+    """
+    query = """
+    PREFIX entity: <http://www.wikidata.org/entity/>
+    #partial results
```

Comment:
Would it be better to put this into `Example` session of the docstring of the function?

Reply:
The function itself is to perform this query with these parameters. If I move the query as an Example, I would need another function to indicate the value of the query
Maybe I'm not understanding it correctly, I would really appreciate it of you could clarify :)

Reply:
i think maybe there was just some confusion as they query is rather complicated it looks like a docstring? I don't think changes are needed here.